### DISCUSSION TITLE
Mac M1

### DISCUSSION THREAD

**Discussion by desistefanova** [crazy-max/ghaction-xgo#309] [Q&A] [answered]
This package is very helpful. I use it in https://github.com/takecontrolsoft/sync_server/pull/8.
Is there any plans for supporting Mac arm architecture builds?

**Comment by crazy-max** [comment, marked as answer]
> Is there any plans for supporting Mac arm architecture builds?

Seems you build for mac arm architecture already in https://github.com/takecontrolsoft/sync_server/blob/77b0aad6af88b9492a9b37e04eb959bde1a3780f/.github/workflows/release.yml#L58 with `darwin/arm64`.

Were you looking for smth else?

And btw thanks a lot for the donation!

**Reply by desistefanova** [reply]
Aah, right! I've missed that. Thanks!
The action is awesome and helped me a lot.
